### ISSUE TITLE
Add option for using subset of training chips

### ISSUE THREAD

**Issue by lewfish** [azavea/raster-vision-fastai-plugin#12]
See https://docs.fast.ai/data_block.html#ItemList.use_partial_data and https://github.com/azavea/raster-vision-fastai-plugin/blob/master/fastai_plugin/semantic_segmentation_backend.py#L179

**Comment by lewfish** [comment]
I had originally wanted to use a subset of the training chips and all of the validation chips, but I don't see an obvious way to do that. But that would be preferable if you can figure it out.

**Comment by simonkassel** [comment]
yeah I ran into the same problem when I was doing my bike rack classifier. I ended up partitioning the dataset into lists of training and validation images, then subsetting training set into the size I wanted. I used https://docs.fast.ai/vision.data.html#ImageDataBunch.create_from_ll to create an ImageDataBunch from two different `LabelList` objects. 

I should be able to do the same here. If we keep the random seed the same we should get the same validation set each time we do the first partition.

**Comment by simonkassel** [comment]
Actually I think it's easier than that. At the point of creating an ImageDataBunch they are already split into folders so I should be able to take a sample of the chips in the train directory, copy them to a new folder, and then pass that folder to `.split_by_folder`. Do you want each subsample of the training set to have the same pixel class distribution?

**Comment by lewfish** [comment]
Good idea. No, you can just use a random subset as long as you seed it.

**Comment by lewfish** [comment]
It less important to balance the classes with the chips because there are so many of them compared with the scenes.
